core/domain/user_services.py

```diff
@@ -1431,32 +1416,6 @@ def update_profile_picture_data_url(
     fs.commit(filename_webp, webp_binary, mimetype='image/webp')
 
 
-def update_user_bio(user_id: str, user_bio: str) -> None:
-    """Updates user_bio of user with given user_id.
-
-    Args:
-        user_id: str. The unique ID of the user.
-        user_bio: str. New user biography to be set.
-    """
-    user_settings = get_user_settings(user_id, strict=True)
-    user_settings.user_bio = user_bio
-    save_user_settings(user_settings)
-
-
-def update_user_default_dashboard(
-    user_id: str, default_dashboard: str
-) -> None:
-    """Updates the default dashboard of user with given user id.
-
-    Args:
-        user_id: str. The unique ID of the user.
-        default_dashboard: str. The dashboard the user wants.
-    """
-    user_settings = get_user_settings(user_id, strict=True)
-    user_settings.default_dashboard = default_dashboard
-    save_user_settings(user_settings)
-
-
 def update_user_creator_dashboard_display(
```

Comment:
Do we need this function either?

Reply:
Actually, this was not used in PreferencesHandler so I have not seen it. But now I found it is not used anywhere in the codebase, so removed it.
Done.